### PR TITLE
Add note about Vite' Pre-bundling to build-a-global-svelte-app blog post

### PR DESCRIPTION
**Pull request by sunxyw**

### CODE REVIEW

**Comment by samuelstroschein** [comment]
@sunxyw `noExternal` is also required to force vite to bundle messages, correct?

File: inlang/guides/build-a-global-svelte-app/build-a-global-svelte-app.md [hunk 1]
```diff
@@ -56,6 +56,30 @@ npm run build
...
+
+Or just in local environment (`serve` or `dev` command):
+```js
+// vite.config.js/ts
+export default defineConfig(({ command }) => ({
+  optimizeDeps: {
+    exclude: command === 'serve' ? ['@inlang/paraglide-js'] : []
+  },
+}));
+```
```
Comment (samuelstroschein) [inline inlang/guides/build-a-global-svelte-app/build-a-global-svelte-app.md:81]: I don't think this is necessary. 

The vite docs mention that optimizeDeps only happens in dev mode. "Dependency pre-bundling only applies in development mode".

Comment (sunxyw) [reply inlang/guides/build-a-global-svelte-app/build-a-global-svelte-app.md:81]: you are right! I'll remove this part.

**Comment by sunxyw** [comment]
> @sunxyw `noExternal` is also required to force vite to bundle messages, correct?

@samuelstroschein I am not sure.

I'm not an expert on vite and don't quite understand what `noExternal` does.
After reading the description in the documentation, this seems to be just an optimization for dependencies that provide both ESM and CommonJS. It should not affect the loading of `messages.js`?

I haven't included `noExternal` in my project but it seems to work.

I'll happily include it in the article if you think it's necessary.

**Review by samuelstroschein — APPROVED** [review]
Seems good. Thanks for the contribution!

**Review by NilsJacobsen — APPROVED** [review]
Looks good to me.

**Pull request merged**